taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/controller/CamundaTaskEventsController.java

```diff
@@ -118,19 +125,22 @@ public Response setRemainingRetries(
   @Consumes(MediaType.APPLICATION_JSON)
   @Produces(MediaType.APPLICATION_JSON)
   public Response setRemainingRetriesForMultipleEvents(
-      @QueryParam("retries") final String retries, Map<String, Integer> newRemainingRetries)
-      throws InvalidArgumentException {
+      @QueryParam("retries") final String retries, String newRemainingRetries)
+      throws InvalidArgumentException, JsonProcessingException {
 
     if (retries == null || retries.isEmpty()) {
       throw new InvalidArgumentException("Please provide a valid \"retries\" query parameter");
     }
 
-    if (newRemainingRetries == null || !newRemainingRetries.containsKey("remainingRetries")) {
+    Map<String, Integer> patchMap =
+        OBJECT_MAPPER.readValue(newRemainingRetries, new TypeReference<Map<String, Integer>>() {});
+
+    if (patchMap == null || !patchMap.containsKey("remainingRetries")) {
       throw new InvalidArgumentException(
           "Please provide a valid json body in the format {\"remainingRetries\":3}");
     }
 
-    int retriesToSet = newRemainingRetries.get("remainingRetries");
+    int retriesToSet = patchMap.get("remainingRetries");
 
     int remainingRetries = Integer.parseInt(retries);
```

Comment:
Why do we use Integer in one service operation an int in another? Not an important point, but we could be easily consistent here.